ECOSYSTEM.md

```diff
@@ -71,7 +71,6 @@ The following are community driven plugins.
 [dokken]: https://github.com/someara/kitchen-dokken
 [lxc]: https://github.com/chrisroberts/kitchen-lxc
 [nodes]: https://github.com/mwrock/kitchen-nodes
-[vsphere]: https://github.com/mwrock/kitchen-vsphere
```

Comment:
Sorry I meant to say we should leave this in but just change url to https://github.com/CenturyLinkCloud/chef-provisioning-vsphere